plugins/discovery/discovery.go

```diff
@@ -255,6 +280,51 @@ func (c *Discovery) reconfigure(ctx context.Context, u download.Update) error {
 	return nil
 }
 
+func (c *Discovery) applyLocalOverride(config *config.Config) error {
+	for name, rawJson := range c.localConficOverrides {
+		var localOverride, target map[string]interface{}
+		util.Unmarshal(rawJson, &localOverride)
```

Comment:
Ok to ignore error?

Reply:
Ahm, good catch. Not probably not, will fix this